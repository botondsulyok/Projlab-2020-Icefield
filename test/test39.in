field spawn1 0
field spawn2 0
entity esk1 Eskimo spawn1
entity esk2 Eskimo spawn2
item shovel Shovel spawn1
item breakable breakable spawn2
stats spawn1
stats spawn2
stats esk1
stats esk2
dig esk1
stats spawn1
stats esk1
dig esk2
stats spawn2
stats esk2