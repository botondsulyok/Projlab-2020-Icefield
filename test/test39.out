id: spawn1
type: iceunit
building: plains0
snow: 0
pickable: shovel
standers: esk1
pullers: esk1
masters: esk1
neighbours: null

id: spawn2
type: iceunit
building: plains1
snow: 0
pickable: breakable
standers: esk2
pullers: esk2
masters: esk2
neighbours: null

id: esk1
type: eskimo
iceUnit: spawn1
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: esk2
type: eskimo
iceUnit: spawn2
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: spawn1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: esk1
pullers: esk1
masters: esk1
neighbours: null

id: esk1
type: eskimo
iceUnit: spawn1
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: shovel
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: spawn2
type: iceunit
building: plains1
snow: 0
pickable: null
standers: esk2
pullers: esk2
masters: esk2
neighbours: null

id: esk2
type: eskimo
iceUnit: spawn2
movebehaviour: walk1
bodyheat: 5
stamina: 3
digger: breakable
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

